english/call-a-doctor/timelord/timelord.md

```diff
@@ -0,0 +1,12 @@
+His eyes widen at the marshmallow hands which have promptly sprouted from the walls upon his arrival and were now reaching for him.  He whips out a gizmo which emits a high-pitched sound, aiming it at various marshmallow hands.  Then he turns to you.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.